context/context.go

```diff
@@ -17,24 +17,14 @@ import (
 // unusually large number of git remotes
 const maxRemotesForLookup = 5
 
-func ResolveRemotesToRepos(remotes Remotes, client *api.Client, base string) (*ResolvedRemotes, error) {
+func ResolveRemotesToRepos(remotes Remotes, client *api.Client) (*ResolvedRemotes, error) {
 	sort.Stable(remotes)
 
 	result := &ResolvedRemotes{
 		remotes:   remotes,
 		apiClient: client,
 	}
 
-	var baseOverride ghrepo.Interface
-	if base != "" {
-		var err error
-		baseOverride, err = ghrepo.FromFullName(base)
-		if err != nil {
-			return result, err
-		}
-		result.baseOverride = baseOverride
-	}
```

Comment:
It seemed like it should not be the responsibility of this function to also handle creating of the repo. So I introduced `SetBaseOverride()` but I would also be okay with this function taking in a `ghrepo.Interface` and just setting it to `baseOverride` if that is preferable.